18 Normalizr/src/model/view/studentView.ts

```diff
@@ -1,13 +1,17 @@
+import { CountryView } from './countryView';
+
 export class StudentView {
   id: number;
   gotActiveTraining: boolean;
   fullname: string;
   email: string;
+  country: CountryView;
 
   public constructor() {
     this.id = -1;
     this.gotActiveTraining = false;
     this.fullname = "";
     this.email = "";
+    this.country = new CountryView();
```

Comment:
Agree, or we could assign a default country entry. Thinking now about pros adding a factory to create then StudentView